scripts/lib/cluster.sh

```diff
@@ -16,15 +16,15 @@ function up-test-cluster() {
     AWS_K8S_TESTER_EKS_NAME=$CLUSTER_NAME \
         AWS_K8S_TESTER_EKS_KUBECONFIG_PATH=$KUBECONFIG_PATH \
         AWS_K8S_TESTER_EKS_KUBECTL_PATH=$KUBECTL_PATH \
-        AWS_K8S_TESTER_EKS_S3_BUCKET_NAME=amazon-vpc-cni-k8s-aws-k8s-tester \
-        AWS_K8S_TESTER_EKS_S3_BUCKET_CREATE=false \
+        AWS_K8S_TESTER_EKS_S3_BUCKET_NAME="" \
+        AWS_K8S_TESTER_EKS_S3_BUCKET_CREATE=true \
```

Comment:
Hmm, in order to be able to override these, we will have to add them as env variables in `run-integration-tests.sh`, just like `$KUBECONFIG_PATH` and `$KUBECTL_PATH` above. It looks like we will need 4 new variables:
```
S3_BUCKET_NAME
S3_BUCKET_CREATE
ROLE_CREATE
ROLE_ARN
```